website/blog/2024-10-07-release-1.13.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Podman Desktop 1.13 Release
+description: Podman Desktop 1.13 has been released!
+slug: podman-desktop-release-1.13
+authors: SoniaSandler
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.13/banner.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.13 Release! 🎉
+
+![podman-desktop-hero-1.13](img/podman-desktop-release-1.13/banner.png)
+
+Podman Desktop 1.13 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **Hyper-V support**: You can now create and manage Hyper-V Podman machines directly from Podman Desktop.
+- **Search image feature**: Search for an image from Podman Desktop.
+- **Updated empty state pages**: Start your journey in the containers, images, pods, and Kubernetes pages with a click of a button
+- **New Kubernetes navigation**: All Kuberenetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
+- **Image Layer explorer extension**: With this new extension, you can explore the various layers of images.
+- **Experimental Docker compatibility mode page**: You can now try out our experimental docker compatibility mode page.
+<!--truncate-->
+
+## Release Details
+
+### Hyper-V support
+
+Podman Desktop is now supporting the creation and modification of Hyper-V based Podman machines in Windows directly from the app.
```

Comment:
Not sure between in Windows or on Windows is better ?

Reply:
on Windows